src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.scss

```diff
@@ -11,3 +11,15 @@
 .btn:disabled {
   pointer-events: none;
 }
+
+.user-warning {
+  color: vv.$danger;
+}
+
+.endpoint-warning {
+  color: vv.$warning;
+}
+
+i {
+  font-size: 1.5em;
+}
```

Comment:
you don't need to use this. we already have pre-defined sizes.

for eg
```
[ngClass]="[icons.stack, icons.large2x,  icons.danger]"
```

Reply:
moreover, for me the current size looks good and fits the rest of the UI. Once you increased the icon it looked a bit weirder.